src/Maui/Shared/MainLayout.razor

```diff
@@ -1,10 +1,10 @@
-﻿@using DMPowerTools.Maui.Features.User;
+﻿@using TabletopPowerTools.Maui.Features.User;
 @implements IDisposable
 @inherits LayoutComponentBase
 
 <MudLayout>
     <MudAppBar Color=Color.Transparent Elevation=0>
-        <MudText Typo="Typo.h3" Color="Color.Primary">DM Power Tools</MudText>
+        <MudText Typo="Typo.h3" Color="Color.Primary">Tabletop Power Tools</MudText>
```

Comment:
In lieu of an icon for now do we just want to abbreviate this to TTPT?

Reply:
Sure, I have no preference there. Changed.